EOS-uiux-Solutions/user-story
# Added product query in the URL

Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwalv@gmail.com>

**Issue Number**

fixes #228 



**Describe the changes you've made**

I have added product query string in the URL  `?product=name` now with this wherever, the productlist string will be used it will update that. On the story page there is product image in that I have added a link that redirects to home with product query.

**Additional context (OPTIONAL)**

I have added query-string package for parsing location search string

**Test plan (OPTIONAL)**

1. Visit home page
2. Click on any story
3. On the story page click the product image
4. Change product on the home page and see URL


**Checklist**
